peril-settings.json

```diff
@@ -63,7 +66,8 @@
                 "Automattic/peril-settings@org/pr/milestone.ts",
                 "Automattic/peril-settings@org/pr/diff-size.ts",
                 "Automattic/peril-settings@org/pr/check-subtrees.ts",
-                "Automattic/peril-settings@org/pr/check-tracks.ts"
+                "Automattic/peril-settings@org/pr/check-tracks.ts",
+                "Automattic/peril-settings@org/pr/release-notes.ts"
```

Comment:
We don't have `RELEASE_NOTES.txt` for `simplenote-ios`, `simplenote-macos` and `simplenote-android`?

Reply:
We do have them in the Simplenote repos, too.